test/bdd/steps/common.mjs

```diff
@@ -301,7 +165,7 @@ Given(
         forkedNode.on('message', (response) => {
             if (response.error) {
                 assert.fail(
-                    `Error while trying initialize node${nodeIndex - 1} client: ${response.error}`,
+                    `Error while trying initialize node${nodeIndex} client: ${response.error}`,
                 );
             } else {
                 // todo if started
```

Comment:
Why do we have this todo here?

Reply:
I think it's been there from before the steps were implemented.Forgot to delete it.